src/server/bootstrap/index.ts

```diff
@@ -72,20 +74,55 @@ export async function bootstrap(): Promise<Express> {
     name: 'checkfirst',
   })
 
+  const sentrySessionMiddleware = (
+    req: Request,
+    _res: Response,
+    next: NextFunction
+  ): void => {
+    if (req.session?.user) {
+      const { email, id } = req.session.user
+      Sentry.setUser({
+        id: id.toString(),
+        email: email,
+      })
+    }
+    next()
+  }
+
   const app = express()
 
+  Sentry.init({
+    dsn: process.env.BACKEND_SENTRY_DSN,
+    // automatically picks up environment from SENTRY_ENVIRONMENT
+    integrations: [
+      // enable HTTP calls tracing
+      new Sentry.Integrations.Http({ tracing: true }),
+      // enable Express.js middleware tracing
+      new Tracing.Integrations.Express({ app }),
+    ],
+    tracesSampleRate: 1.0,
```

Comment:
... by injecting it as `environment:` here, ie 
```suggestion
    tracesSampleRate: 1.0,
    environment: config.get('nodeEnv'),
```